app/uk/gov/hmrc/gform/gform/FormController.scala

```diff
@@ -316,10 +316,17 @@ class FormController(
 
       validateAndUpdateData(cacheUpd, processDataUpd, sn, sn, maybeAccessCode, ff, formModelOptics) { _ =>
         val sectionTitle4Ga = getSectionTitle4Ga(processDataUpd, sn)
-        Redirect(
-          routes.FormController
-            .form(cache.formTemplate._id, maybeAccessCode, sn, sectionTitle4Ga, SuppressErrors.Yes, FastForward.Yes)
-        )
+
+        if (idx === 0)
```

Comment:
Is this logic right? idx could be 0 even when there are 2 items and the first one is being removed